src/index.ts

```diff
@@ -1,8 +1,7 @@
-
 export { default as EncryptedFS } from './EncryptedFS';
+export { default as Cryptor } from './Crypto';
+
+export { Stat } from './util'
 
-// polyfills to be exported
-// $FlowFixMe: Buffer exists
 export { Buffer } from 'buffer';
-// $FlowFixMe: nextTick exists
-export { nextTick } from 'process';
\ No newline at end of file
+export { nextTick } from 'process';
```

Comment:
If you are not using `nextTick`, then this must be removed. And the `process` library as well.

Reply:
And since you are propagating the lowerfs async, you don't need to implement your own async wrapper, so we shouldn't need to have a `nextTick`.

Reply:
Propagating the lowerfs is `promisify`.

Reply:
callbacks have been promisified and we are now using `async` implicit promises in lieu of `nextTick`